arch/arm/configs/aspeed_defconfig

```diff
@@ -148,7 +151,7 @@ CONFIG_USB_STORAGE=y
 CONFIG_USB_SERIAL=y
 CONFIG_USB_SERIAL_GENERIC=y
 CONFIG_USB_GADGET=y
-CONFIG_USB_G_SERIAL=y
+CONFIG_USB_G_SERIAL=m
 CONFIG_MMC=y
```

Comment:
Hey,

On Mon, 12 Oct 2015 23:06:54 Joel Stanley wrote:
> > @@ -148,7 +151,7 @@ CONFIG_USB_STORAGE=y
> >  CONFIG_USB_SERIAL=y
> >  CONFIG_USB_SERIAL_GENERIC=y
> >  CONFIG_USB_GADGET=y
> > -CONFIG_USB_G_SERIAL=y
> > +CONFIG_USB_G_SERIAL=m
> >  CONFIG_MMC=y
> 
> What's up with this change?

Just talked Cyril ... no reason really. Apparently the kernel defconfig generator made the change. Is this something we care about?

> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openbmc/linux/pull/2/files#r41829799